.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: c
+dist: trusty
+sudo: required
+
+notifications:
+  email: false
+
+git:
+  submodules: true
+  depth: 3
+
+matrix:
+  include:
+    - os: linux
+      compiler: gcc
+    - os: osx
+      compiler: clang
+
+script:
+  - make UTIL=true EXTRA=true MULTI=true
+  - cd samples/c/pdtest && make && cd ../../..
+  - cd samples/c/pdtest_gui && make && cd ../../..
+  - cd samples/c/pdtest_multi && make && cd ../../..
+  #- cd samples/c/pdtest_thread && make && cd ../../..
+
+after_success:
+  - cd samples/c/pdtest && ./pdtest test.pd ./ && cd ../../..
+  #- cd samples/c/pdtest_gui && ./pdtest_gui test.pd ./ && cd ../../..
+  - cd samples/c/pdtest_multi && ./pdtest_multi test.pd ./ && cd ../../..
+  #- cd samples/c/pdtest_thread && pdtest_thread ./ test.pd && cd ../../..
```

Comment:
Note: I just pushed a commit which changes pdtest_thread to use the same file & folder arguments as the other C samples. I was having trouble with the paths before.

Reply:
Thanks ! I was correcting it  also :) I keep you informed when the branch is ready (there is a problem with pthread on Linux)